File: src/naive/JavaScript/fib.js
const fib = n => {
	if (n < 3) {
        return 1;
    } else {
        return fib(n - 1) + fib(n - 2);
    }
}

const main = () => {
    const num = parseInt(process.argv[2])
    const result = fib(num);
    console.log(`${result}`);
}

main()

File: src/naive/Python/Python/fib.py
"""
Naive implementation of fibbonaci sequence
"""
import sys


def fib(num):
    """Return nth fibonacci number"""
    return 1 if num < 3 else fib(num-1) + fib(num-2)


def main():
    """Entry poing"""
    num = int(sys.argv[1])
    print("{}".format(fib(num)))


if __name__ == "__main__":
    main()
